projects/storefrontlib/src/cms-components/navigation/search-box/search-box.component.html

```diff
@@ -7,7 +7,11 @@
     (input)="search(searchInput.value)"
     (blur)="close($event)"
     (keydown.escape)="close($event)"
-    (keydown.enter)="launchSearchResult($event, searchInput.value)"
+    (keydown.enter)="
+      launchSearchResult($event, searchInput.value); close($event, true)
```

Comment:
close is called twice here.
leave `launchSearchResult($event, searchInput.value);`